spring-integration-core/src/main/java/org/springframework/integration/dsl/MessageChannels.java

```diff
@@ -101,27 +101,24 @@ public static PriorityChannelSpec priority(PriorityCapableChannelMessageStore me
 	}
 
 	public static PriorityChannelSpec priority(String id,
-			PriorityCapableChannelMessageStore messageGroupStore, Object groupId) {
+											   PriorityCapableChannelMessageStore messageGroupStore, Object groupId) {
 		return priority(messageGroupStore, groupId).id(id);
```

Comment:
Checkstyle doesn't allow this style of indentation; will fix on merge.

Reply:
Huh? Looks like this is an artifact after upgrading to a new IDEA. Sorry

I believe we need to start moving to something like https://github.com/spring-io/spring-javaformat/ ...

Thanks BTW

Reply:
(Fixed import order too).

Reply:
I don't like the javaformat line limit

>The source formatter intentionally uses a low character count of 90 chars for wrapping. If you’re used to longer lines, this can take some getting used to. Specifically, if you have many nesting levels things can start to look quite bad.